src/java/us/kbase/mobu/tester/SDKSubsequentCallRunner.java

```diff
@@ -36,6 +40,43 @@ public SDKSubsequentCallRunner(
     }
 
     @Override
+    protected ModuleVersion loadModuleVersion(ModuleMethod modmeth,
+            String serviceVer) throws IOException, JsonClientException {
+        if (isLocalModule(modmeth.getModule())) {
+            return new ModuleVersion().withModuleName(modmeth.getModule())
+                    .withGitUrl("http://localhost")
+                    .withGitCommitHash("local-docker-image").withVersion("local");
+        } else {
+            return super.loadModuleVersion(modmeth, serviceVer);
+        }
+    }
+
+    protected Map<String, String> getLocalModuleNameToImage() {
+        return null;
+    }
```

Comment:
Should note in comment that this method is intended for override to provide local module image names.